tasks/build-versions-comparison.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Build the versions comparison page
+ */
+const gulp = require('gulp');
+const hbs = require('handlebars');
+const gulpHandlebars = require('gulp-handlebars-html')(hbs);
+const fs = require('fs');
+const rename = require('gulp-rename');
+const revReplace = require('gulp-rev-replace');
+const _ = require('lodash');
+
+const majorVersion = 'serenity';
+const features = [
+    'Asset',
+    'Reference entities',
+    'Rules engine'
+];
+const domainColors = {
```

Comment:
I would directly put the correct format in the const for clarity, as you use it just once and transform it in another format:

```
const coloredDomains = {
   { 'color' : 'info', 'domain': 'Productivity'},
   ....
};
```

Reply:
👍